addon/services/intl.js

```diff
@@ -38,7 +39,7 @@ export default Service.extend(Evented, {
 
       if (!isArrayEqual(proposed, this._locale)) {
         this._locale = proposed;
-        this.trigger('localeChanged');
+        next(() => this.trigger('localeChanged'));
```

Comment:
Can you help me understand this issue of needing to schedule on the next runloop.  `localeChanged` is relied on in the helpers to recompute on locale change.  Is that behavior broken?

https://github.com/ember-intl/ember-intl/blob/89009718cb36ebfce0b2783e9aff05702d9f6431/addon/helpers/-format-base.js#L22